src/Rtc/Create.tsx

```diff
@@ -124,7 +128,53 @@ const Create = ({
     return () => {
       engine.current!.destroy();
     };
-  }, []);
+  }, [rtcProps.appId]);
+
+  useEffect(() => {
```

Comment:
You need to have an internal ref to track the `enableVideo` state. I see it being enabled if you are Host. But it is never disabled for Attendee

Reply:
So if you go from Host to Attendee to Host, It will be enabled twice without disabling first